### PR TITLE
biber{,-ms}: misc biber fixes

### PR DESCRIPTION
**Pull request by xworld21**
## Description of changes
Remove outdated package from biber, update its license to use the one provided by the texlive database, add new biber-ms package.

## Things done
- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [X] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/typesetting/biber-ms/default.nix [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    mv "$out"/bin/biber{,-ms}
+  '' + lib.optionalString stdenv.isDarwin ''
+    shortenPerlShebang "$out"/bin/biber-ms
+  '';
+
+  meta = with lib; {
+    description = "Backend for BibLaTeX (multiscript version)";
+    license = biberSource.meta.license;
+    platforms = platforms.unix;
+    maintainers = [ maintainers.xworld21 ];
```
Comment (xworld21) [inline pkgs/tools/typesetting/biber-ms/default.nix:50]: @ttuegel do you want to be maintainer for biber-ms?

**Review by veprbl — DISMISSED** [review]

**Comment by xworld21** [comment]
@ofborg build biber-ms

**Comment by xworld21** [comment]
>        error: undefined variable 'LWPUserAgent'

I think I get the pattern: in `perl-packages.nix`, some packages are declared as aliases, e.g. `TestMore = self.TestSimple` and `LWPUserAgent = self.LWP`. Any idea why this causes an evaluation error?

**Comment by xworld21** [comment]
image
Took me a few tries! Next time I'll override `config.allowAliases` locally.

**Review by apfelkuchen6 — APPROVED** [review]
This fixes the licensing issues uncovered in #247798. I haven't tested it, but the diff looks fine.

**Pull request merged**